<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1062</width>
    <height>776</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(81, 0, 135, 255), stop:0.427447 rgba(41, 61, 132, 235), stop:1 rgba(155, 79, 165, 255));
font-family: Century Gothic;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame_3">
      <property name="styleSheet">
       <string notr="true">background-color: none;
border: 0
</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QFrame" name="kbsh">
           <property name="styleSheet">
            <string notr="true">background-color: rgba(255,255,255,30%);
border: 1px solid rgba(255,255,255,40%);
border-radius: 7px;</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QFrame" name="frame_kbsh">
              <property name="styleSheet">
               <string notr="true">background-color: none;
border: 0
</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QLabel" name="le_all">
                 <property name="styleSheet">
                  <string notr="true">color: white;
font-weight: bold;
font-size: 20pt;
background-color: none;
border:  none;
</string>
                 </property>
                 <property name="text">
                  <string>КБШ НТЭ</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QFrame" name="all_type">
                 <property name="styleSheet">
                  <string notr="true">background-color: none;
border: none
</string>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout">
                  <property name="sizeConstraint">
                   <enum>QLayout::SetDefaultConstraint</enum>
                  </property>
                  <item>
                   <layout class="QVBoxLayout" name="type">
                    <item>
                     <widget class="QLabel" name="le_mulcher">
                      <property name="styleSheet">
                       <string notr="true">color: white;
font-weight: bold;
font-size: 16pt;
background-color: none;
border:  none;
</string>
                      </property>
                      <property name="text">
                       <string>Мульчер</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="le_ech">
                      <property name="styleSheet">
                       <string notr="true">color: white;
font-weight: bold;
font-size: 16pt;
background-color: none;
border:  none;
</string>
                      </property>
                      <property name="text">
                       <string>Экспл. персонал</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="le_davs">
                      <property name="styleSheet">
                       <string notr="true">color: white;
font-weight: bold;
font-size: 16pt;
background-color: none;
border:  none;
</string>
                      </property>
                      <property name="text">
                       <string>ДАВС</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="le_pod">
                      <property name="styleSheet">
                       <string notr="true">color: white;
font-weight: bold;
font-size: 16pt;
background-color: none;
border:  none;
</string>
                      </property>
                      <property name="text">
                       <string>Подрядные организации</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QVBoxLayout" name="plan">
                    <item>
                     <widget class="QLabel" name="le_plan_mulcher">
                      <property name="styleSheet">
                       <string notr="true">color: white;
font-weight: bold;
font-size: 16pt;
background-color: none;
border:  none;
</string>
                      </property>
                      <property name="text">
                       <string>200</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="le_plan_ech">
                      <property name="styleSheet">
                       <string notr="true">color: white;
font-weight: bold;
font-size: 16pt;
background-color: none;
border:  none;
</string>
                      </property>
                      <property name="text">
                       <string>200</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="le_plan_davs">
                      <property name="styleSheet">
                       <string notr="true">color: white;
font-weight: bold;
font-size: 16pt;
background-color: none;
border:  none;
</string>
                      </property>
                      <property name="text">
                       <string>200</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="le_plan_pod">
                      <property name="styleSheet">
                       <string notr="true">color: white;
font-weight: bold;
font-size: 16pt;
background-color: none;
border:  none;
</string>
                      </property>
                      <property name="text">
                       <string>200</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QVBoxLayout" name="fact">
                    <item>
                     <widget class="QLabel" name="le_fact_mulcher">
                      <property name="styleSheet">
                       <string notr="true">color: white;
font-weight: bold;
font-size: 16pt;
background-color: none;
border:  none;
</string>
                      </property>
                      <property name="text">
                       <string>200</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="le_fact_ech">
                      <property name="styleSheet">
                       <string notr="true">color: white;
font-weight: bold;
font-size: 16pt;
background-color: none;
border:  none;
</string>
                      </property>
                      <property name="text">
                       <string>200</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="le_fact_davs">
                      <property name="styleSheet">
                       <string notr="true">color: white;
font-weight: bold;
font-size: 16pt;
background-color: none;
border:  none;
</string>
                      </property>
                      <property name="text">
                       <string>200</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="le_fact_pod">
                      <property name="styleSheet">
                       <string notr="true">color: white;
font-weight: bold;
font-size: 16pt;
background-color: none;
border:  none;
</string>
                      </property>
                      <property name="text">
                       <string>200</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QFrame" name="btn_all">
         <property name="styleSheet">
          <string notr="true">background-color: none;
</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="btn_add">
            <property name="styleSheet">
             <string notr="true">QPushButton {
color: white;
background-color: rgba(255,255,255,30);
border: 1px solid  rgba(255,255,255,40);
border-radius: 7px;
width: 230px;
height: 50px;
}
QPushButton:hover {
background-color: rgba(255,255,255,40);
}
QPushButton:pressed {
background-color: rgba(255,255,255,70);
}</string>
            </property>
            <property name="text">
             <string>Добавить участок</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_edit">
            <property name="styleSheet">
             <string notr="true">QPushButton {
color: white;
background-color: rgba(255,255,255,30);
border: 1px solid  rgba(255,255,255,40);
border-radius: 7px;
width: 230px;
height: 50px;
}
QPushButton:hover {
background-color: rgba(255,255,255,40);
}
QPushButton:pressed {
background-color: rgba(255,255,255,70);
}</string>
            </property>
            <property name="text">
             <string>Изменить участок</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_delete">
            <property name="styleSheet">
             <string notr="true">QPushButton {
color: white;
background-color: rgba(255,255,255,30);
border: 1px solid  rgba(255,255,255,40);
border-radius: 7px;
width: 230px;
height: 50px;
}
QPushButton:hover {
background-color: rgba(255,255,255,40);
}
QPushButton:pressed {
background-color: rgba(255,255,255,70);
}</string>
            </property>
            <property name="text">
             <string>Удалить участок</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="tableView">
      <property name="styleSheet">
       <string notr="true">QTableView {
background-color: rgba(255,255,255,30);
border: 1px solid  rgba(255,255,255,40);
border-bottom-right-radius: 7px;
border-bottom-left-radius: 7px;
}

QTableView::section {
background-color: rgba(53,53,53);
color: white;
borber: none;
height: 50px;
font-size: 14pt;
}
QTableView::item {
border-style: none;
border-bottom: rgba(255,255,255,30); 
}

QTableView::item:select {
border: none;
color: rgba(255,255,255);
background-color: rgba(255,255,255,50);
}
</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
